bob-access/src/extractor.rs

```diff
@@ -50,21 +49,21 @@ impl<T> Extractor<Request<T>> for BasicExtractor {
     }
 }
 
-impl<'r> Extractor<RRequest<'r>> for BasicExtractor {
-    fn extract(&self, req: &RRequest) -> Result<Credentials, Error> {
-        let headers = req.headers();
-        let username = headers
-            .get_one("username")
-            .ok_or_else(|| Error::credentials_not_provided("username"))?;
-        let password = headers
-            .get_one("password")
-            .ok_or_else(|| Error::credentials_not_provided("password"))?;
-        Ok(Credentials::builder()
-            .with_username_password(username, password)
-            .with_address(req.remote())
-            .build())
-    }
-}
+// impl<'r> Extractor<RRequest<'r>> for BasicExtractor {
```

Comment:
Do I understand correctly, that this will work through `impl<T> Extractor<Request<T>>`?

Reply:
It's impl for rocket request. We moved to axum.